pages/common/wami.md

```diff
@@ -0,0 +1,20 @@
+# wami
+
+> A tool that recommends suitable programs for specific tasks. Open-source, easy to use.
+> More information: <https://github.com/evait-security/wami>.
+
+- Search the lake and find expanded results in all categories in descending order:
+
+`wami --show-[a]ll --[S]ort desc --[s]earch-all {{search_string}}`
```

Comment:
@sebastiaanspeck @kbdharun the mnemonics should stay in the commands or in the description?

Reply:
The descriptions as far I know.